docs/USER_GUIDE.md

```diff
@@ -195,6 +195,9 @@ The Maestro Command Line Interface (CLI) allows users to manage workflows that i
   - `--host HOST`: host to bind to (default: 127.0.0.1)
   - `--agent-name NAME`: specific agent name to serve (if multiple agents in file)
   - `--streaming`: enable streaming responses
+- `maestro serve-workflow` AGENTS_FILE WORKFLow_FILE [options]: serve workflow via HTTP API endpoints
```

Comment:
would it be possible to do like with `maestro run` and be able to pass either (1) just an agents file, or (2) both an agents file and workflow file? 

(I think it's a nicer UX to have just one command, even if we keep separate functions for both patterns behind the scenes)

Reply:
I'll try it.  Thanks!